packages/emails/templates/organizer-scheduled-email.ts

```diff
@@ -120,17 +121,34 @@ ${callToAction}
     return this.calEvent.organizer.timeZone;
   }
 
+  protected getLocale(): string {
+    return this.calEvent.organizer.language.locale;
+  }
+
   protected getOrganizerStart(format: string) {
-    return this.getRecipientTime(this.calEvent.startTime, format);
+    return this.getFormattedRecipientTime({
+      time: this.calEvent.startTime,
+      format,
+      locale: this.getLocale(),
+      timeZone: this.getTimezone(),
+    });
   }
 
   protected getOrganizerEnd(format: string) {
-    return this.getRecipientTime(this.calEvent.endTime, format);
+    return this.getFormattedRecipientTime({
+      time: this.calEvent.endTime,
+      format,
+      locale: this.getLocale(),
+      timeZone: this.getTimezone(),
+    });
   }
 
   protected getFormattedDate() {
-    return `${this.getOrganizerStart("h:mma")} - ${this.getOrganizerEnd("h:mma")}, ${this.t(
```

Comment:
getFormattedDate seems to be using the same logic in all classes. Maybe we can extract out the logi in a `getFormattedDateRange(start, end)` kind of thing.

Suitable for a followup I guess.

Reply:
also this function maybe be moved to **packages/lib/timeFormat.ts**

Reply:
okay will do this in a follow up PR